hermeznetwork/bridge-ui
# Improve routing

### What does this PR does?

This PR improves the routing of the app.

### How to test?

- Try to access to a non existing route. You should be redirected to the `/` and then to the `/login` as you won't be logged in.
- Try to access to the `localhost:3000/activity`. You should be redirected to the `/login` and after logging in you should be redirected to the original route: `/activity.`

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
